vignettes/articles/json.Rmd

```diff
@@ -1,14 +1,15 @@
 ---
```

Comment:
Should the `.Rmd` files be placed in `vignettes/FILE.Rmd`? (Not `vignettes/articles/FILE.Rmd`)

After pkgdown make the articles, I believe the current url will be something like `articles/articles/FILE.html`.

If `vignettes/FILE.Rmd` works the same, please move the files accordingly.

Reply:
... Wait... If they are under the articles folder, then the vignette will not be present in the package, but will be present in the pkgdown articles?

If so, TIL and this is great to leave as is!

Reply:
Indeed, I only left the intro one as a vignette, because it is pretty light (no image or embedded maps_

Reply:
TIL! Thank you!